proto-build/src/transform.rs

```diff
@@ -151,12 +154,14 @@ fn transform_items(
                 let s = transformers::add_derive_eq_struct(&s);
                 let s = transformers::append_attrs_struct(src, &s, descriptor);
                 let s = transformers::serde_alias_id_with_uppercased(s);
+                let s = transformers::respect_gogoproto_nullable(s, descriptor);
```

Comment:
since structures are always optional as a result of rust proto generation (e.g. [this field is not supposed to be optional](https://github.com/neutron-org/neutron-std/blob/dd2927b127becb05fa2a1d154e61a4ec4fd35a0d/packages/neutron-std/src/types/neutron/interchainqueries.rs#L243)), I wonder if it's possible to do the opposite: make structs optional only if there is a gogoproto_nullable set, and make them required otherwise?

Reply:
I'm sure it's possible, but it's out of scope. Could you create a task for that?

Reply:
sure, will do